velox/docs/functions/spark/math.rst

```diff
@@ -203,6 +203,10 @@ Mathematical Functions
 
     Returns the square root of ``x``.
 
+.. spark:function:: cbrt(x) -> double
```

Comment:
Apology for the delayed notice, but the documentation should be added following the alphabetical order. I suggest fixing this in a later patch.